includes/application.php

```diff
@@ -1,4 +1,5 @@
 <?php
+	$APPLICATION_DIR = $APPLICATION_DIR ?? str_replace('includes', '', dirname(__FILE__));
 	spl_autoload_register(function ($class_name) {
 		global $APPLICATION_DIR;
```

Comment:
According to PSR2 standards, opening braces of a method MUST go onto the next line. This should hold for functions also, can you please check and change accordingly.